addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -161,10 +161,12 @@ ul.o_checklist {
         }
     }
 }
-ul.o_checklist[dir="rtl"] > li:not(.oe-nested)::before {
-    left: auto;
-    right: - $o-checklist-margin-left;
-    text-align: right;
+ul.o_checklist[dir="rtl"], ul.o_checklist[dir="ltr"] {
```

Comment:
Why apply the style to the LTR case though ?

Reply:
He added LTR due to RTL languages. See this [video](https://drive.google.com/file/d/1FBVYHZpDkX-RB4-ypNSLLGke3D5ytICE/view) from Testing IN.

Then the question might be why the same styles?
Because `rtlcss` changes the left and right accordingly
Image1: Case for rtl in ltr language
![image](https://github.com/odoo/odoo/assets/120459801/349db43a-464a-485a-96a9-34dd5f8459d0)
Image2: Case for ltr in rtl language
![image](https://github.com/odoo/odoo/assets/120459801/046cfdee-e7ed-42e6-8564-019b627662bb)

Reply:
Okay, I didn't know about rtlcss, but then why do we need `ul.o_checklist[dir="rtl"], ul.o_checklist[dir="ltr"]` ? What happens if we use only `ul.o_checklist`? 

And what happens if we create a checklist in RTL language that is switched back to LTR then we open the document with the forced LTR checklist on another user which uses a LTR language?

Reply:
> What happens if we use only ul.o_checklist?

Its css is already defined [above](https://github.com/odoo/odoo/blob/15.0/addons/web_editor/static/src/scss/web_editor.common.scss#L142). Here we only want to handle lists with dir attributes.

> And what happens if we create a checklist in RTL language that is switched back to LTR then we open the document with the forced LTR checklist on another user which uses a LTR language?

We missed that case.^^'

I have updated the code on behalf of @adch-odoo. He is on leave for couple of weeks.

Reply:
I tested on runbot but I can still end up with a wrongly displayed checklist if I create a checklist then switch to a rtl language. The checkboxes are correctly displayed on the right but the text ends up on the left.

Reply:
>  if I create a checklist then switch to a rtl language.

https://drive.google.com/file/d/10iDnh3-zZbOpPJ4igOwr56nRW_83-Hey/view?usp=drivesdk am I missing anything?

Reply:
Well, somehow it seems to work in Notes yes, but it doesn't work in project, see: https://63254907-15-0-all.runbot200.odoo.com/web#id=74&menu_id=527&cids=1&action=707&active_id=14&model=project.task&view_type=form

Reply:
@dmo-odoo @deso-odoo I've found the steps to reproduce:
Create checklist, switch direction (adds text-align: right), switch direction again (adds text-align: left). Now change to a RTL language.
The issue occurres because when changing the language `text-align: left` is not converted to `text-align: right;`  I can even produce it without a check list (https://www.awesomescreenshot.com/video/28492522?key=03e2e3c9330917df0daa248bfd67f46c) the issue seems to come from https://github.com/odoo/odoo/blob/15.0/addons/web_editor/static/lib/odoo-editor/src/commands/commands.js#L437 instead of using inline style if I use class like `text-left` and `text-right` it works fine.

Reply:
@adch-odoo Nice catch! Could you add a commit to this PR to fix that particular issue? Thanks!